File: caesar cipher/mine.py
def caesarCipher(s, k):
    s=list(s)
    for i in range(len(s)):
        if s[i].isalpha():
            caesar_ascii = ord(s[i])+(k%26)
            if s[i].islower():
                if caesar_ascii > 122:
                    caesar_ascii-=26
            else:
                if caesar_ascii > 90:
                    caesar_ascii-=26
            s[i]=chr(caesar_ascii)
        print("".join(s))
    return ''.join(s)

File: caesar cipher/mine.go
func caesarCipher(s string, k int32) string {
    var output = ""
    var caesar_ascii rune
    var chr string
    for _,char := range s{
        chr = string(char)
        if(unicode.IsLetter(char)){
            caesar_ascii = char+(k%26)
            if(unicode.IsLower(char)){
                if caesar_ascii > 122{
                    caesar_ascii-=26
                }
            }else{
                if caesar_ascii > 90{
                    caesar_ascii-=26
                }
            }
            chr = string(caesar_ascii)
        }
        output+= chr
    }
    return output